INSTALL.md

```diff
@@ -164,6 +164,18 @@ Then you can try one of the
 [examples](https://github.com/kubernetes/client-go/tree/v2.0.0/examples/) from
 the 2.0.0 release.
 
+For client-go 3.0.0 must be defined `k8s.io/apimachinery` override in Gopkg.toml
+
+```
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "3.0.0"
+
+[[override]]
+  name = "k8s.io/apimachinery"
+  branch = "release-1.6"
```

Comment:
It only works with client-go v4.0.0.

Actually could constraint apimachinery to commit 1fd2e63a9a370677308a42f24fd40c86438afddf? That's what godep suggests: https://github.com/kubernetes/client-go/blob/v4.0.0/Godeps/Godeps.json#L336